packages/tailwindcss-language-server/src/project-locator.ts

```diff
@@ -341,6 +342,9 @@ export class ProjectLocator {
     // Resolve real paths for all the files in the CSS import graph
     await Promise.all(imports.map((file) => file.resolveRealpaths()))
 
+    // Resolve all @source directives
+    await Promise.all(imports.map((file) => file.resolveSourceDirectives()))
```

Comment:
@thecrypticace I moved this to the "resolve" step now like we discussed in our call last week.